### ISSUE TITLE
Number of Nodes in the pre-training embedding file

### ISSUE THREAD

**Issue by psycoplankton** [hwwang55/GraphGAN#30]
It seems that the number of nodes for which the node-embeddings are realized, are not equal to the number of nodes of the network. This is causing the accuracy to be lower than what is mentioned in the paper. If anyone has used DeepWalk to generate their own embeddings, could you share them with me? @hwwang55
